README.md

```diff
@@ -181,8 +180,6 @@ PyQuil is licensed under the
 [conda-forge-repo]: https://anaconda.org/conda-forge/pyquil
 [conda-rigetti-badge]: https://img.shields.io/conda/vn/rigetti/pyquil?label=conda-rigetti
 [conda-rigetti-repo]: https://anaconda.org/rigetti/pyquil
-[coverage-badge]: https://coveralls.io/repos/github/rigetti/pyquil/badge.svg?branch=more-badges
-[coverage-repo]: https://coveralls.io/github/rigetti/pyquil?branch=more-badges
```

Comment:
Any reason for not continuing to use coveralls?

Reply:
It was really only ever used for [one build](https://coveralls.io/github/rigetti/pyquil) on that `more-badges` branch in 2019, so we technically used it, but not really. No one seemed to have a strong preference for a third party service, and I've spent too much time debugging problems with them (not coveralls specifically, but a couple of other ones) on other projects, so I thought I'd keep it simple here since the feature has been dead for 2.5 years. If it turns out to be really valuables I'd be happy to invest more time in a fancier solution.

Reply:
Didn't realize we weren't actually using it in that way. The change makes sense, then.